### ISSUE TITLE
Can image be mapped to img?

### ISSUE THREAD

**Issue by apulverizer** [watir/nerodia#15]
## Meta -
Nerodia Version: 0.9.0
Selenium Version: 3.14.0
Browser Version: Chrome
Browser Driver Version: ChromeDriver 2.41.57870
OS Version: MacOS 10.14

## Expected Behavior -
I can find an `image` element and do something with it. Specifically I'm trying to find the number of images in an svg
`len(self.browser.elements(xpath=f"//div[@id='map_layers']//*[@id='{layer_id}']/*[contains(local-name(), 'image')]"))` 

## Actual Behavior -
I cannot find an `image` element because the tag does not seem to be supported/mapped. And this blows up in `element_collection.py` because `kls` is None as it's not in the dictionary.

```python
kls = nerodia.element_class_for(tag_name)
new_selector = dict(self.selector, element=e, tag_name=tag_name, index=dic[tag_name] - 1)
yield kls(self.query_scope, new_selector)
```

I'm able to work around it by adding a custom mapping as shown below:
`nerodia.tag_to_class['image'] = nerodia.elements.image.Image`

I'm wondering if this could be added to the default mappings (happy to do the PR) even though `image` technically isn't a valid html tag despite being used quite frequently.

**Comment by lmtierney** [comment]
I'll have to look into this. I believe we support custom tag names so it may be a bug. It shouldn't map to `Image` though unless it's a valid html tag

**Comment by lmtierney** [comment]
Oh yes, this was fixed in `0.10.0`. Can you update and try?

**Comment by apulverizer** [comment]
Yeah sure, will give a spin. Thanks for the quick response.

**Comment by apulverizer** [comment]
Fixed by upgrading to `0.10.0`. Thanks.
